### DISCUSSION TITLE
[RFC]: 关于私有化部署集群的建议

### DISCUSSION THREAD

**Discussion by crixusshen** [web-infra-dev/modern.js#4460] [General]
### 这个功能解决了什么问题？

由于modern涉及到运维服务器这部分，对开发者而言比较黑盒，但是它确实非常重要的一块。想咨询下如果要进行私有化本地服务器部署，在集群上有什么建议和方案，可以实现高并发高可用？

### 你期望的 API 是什么样子的？

同上

**Comment by zllkjc** [comment]
这块我们也在做考虑，在 9 月会有一些方案提供，目前还是通过 start 命令来完成部署

**Comment by crixusshen** [comment]
> 这块我们也在做考虑，在 9 月会有一些方案提供，目前还是通过 start 命令来完成部署

期待，希望快点更新，目前已经很完善，只要增强生产实施这块的方案就完美了
